src/components/NodeToolbar/Photos/PhotoNotification.js

```diff
@@ -0,0 +1,130 @@
+// @flow
+
+import styled from 'styled-components';
+import * as React from 'react';
+import colors from '../../../lib/colors';
+import { t } from 'c-3po';
+import CheckmarkIcon from '../../icons/actions/CheckmarkIcon';
+import ProblemIcon from '../../icons/actions/ProblemIcon';
+
+type Props = {
+  className: string,
+  notificationType: 'uploadProgress' | 'uploadFailed' | 'reported' | 'waitingForReview';
+  uploadProgress?: number,  // between 0 and 100
+};
+
+type State = {
+};
```

Comment:
I guess this object is not needed :)

Reply:
Tom created lots of boiler plate, I will clean up the remainders.